cmd/token_test.go

```diff
@@ -89,6 +89,43 @@ func TestTokenWithGCEAK(t *testing.T) {
 	}
 }
 
+func TestCustomEventLogFile(t *testing.T) {
+	ExternalTPM = nil
```

Comment:
So in a test, you probably need to use
```
rwc := test.GetTPM(t)
defer client.CheckedClose(t, rwc)
ExternalTPM = rwc
```
to get a TPM

Reply:
Thanks for the comment! This test is not trying to test with the simulator. This test is to test the eventlog flag. And this test is expected to open /dev/tpm0 and /test-event-log. So this test is expected to return failed to open /dev/tpm0 error for environments with no TPM and failed to open /test-event-log for environments with a TPM. Please let me know if this makes sense!